src/common/mmo.h

```diff
@@ -479,8 +480,9 @@ struct status_change_data {
 };
 
 struct storage_data {
-	int storage_amount;
-	struct item items[MAX_STORAGE];
+	bool save; //< save flag.
+	uint32 aggregate; //< total item count.
```

Comment:
Can you make this a signed variable instead? I don't really want to see any unsigned used for any reason other than bitmasks or shifts (unless it's something that the client forces us to do)